policies/WindowsTerminal.admx

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--  (c) 2024 Microsoft Corporation  -->
+<policyDefinitions xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" revision="1.0" schemaVersion="1.0" xmlns="http://schemas.microsoft.com/GroupPolicy/2006/07/PolicyDefinitions">
+  <policyNamespaces>
+    <target prefix="terminal" namespace="Microsoft.Policies.WindowsTerminal" />
+    <using prefix="windows" namespace="Microsoft.Policies.Windows" />
+  </policyNamespaces>
+  <resources minRequiredRevision="1.0" />
+  <categories>
+    <category name="WindowsTerminal" displayName="$(string.WindowsTerminal)">
+      <parentCategory ref="windows:WindowsComponents" />
+    </category>
+  </categories>
+  <policies>
+    <policy name="DisabledProfileSources" class="Both" displayName="$(string.DisabledProfileSources)" explainText="$(string.DisabledProfileSourcesText)" presentation="$(presentation.DisabledProfileSources)" key="Software\Policies\Microsoft\Windows\Terminal">
+      <parentCategory ref="WindowsTerminal" />
+      <supportedOn ref="windows:SUPPORTED_Windows10_0_20H1" />

```

Comment:
Please create your own string. Because the functionality of the policy depends on the WT version and not on the os version.

Reply:
Thanks! I'll look into how that's done.

Reply:
Similar to class definition.

Reply:
we will need to figure out what to do when this version of terminal is pulled into Windows, too...

Reply:
I guess we can just keep it as "At least Windows Terminal 1.21" given that the next time we can raise the inboxed version will be far in the future anyway.